cwf/cloud/go/services/cwf/obsidian/handlers/handlers.go

```diff
@@ -356,7 +356,7 @@ func createHAPairHandler(c echo.Context) error {
 	if err := c.Bind(haPair); err != nil {
 		return obsidian.HttpError(err, http.StatusBadRequest)
 	}
-	if err := haPair.ValidateModel(); err != nil {
+	if err := haPair.ValidateModel(context.Background()); err != nil {
```

Comment:
Why context.Background here? Assuming we can grab the reqCtx from the echo.Context and pass it in